### ISSUE TITLE
Issue to check with Tinkers?

### ISSUE THREAD

**Issue by tgstyle** [The-Fireplace-Minecraft-Mods/Mob-Rebirth#7]
This happened right as the zombies died. Could it be something Mob-Rebirth is causing?

https://github.com/SlimeKnights/TinkersConstruct/issues/2652

Version 7.0.2 for MC 1.10.2

**Comment by The-Fireplace** [comment]
I doubt it, unless Tinker's is doing something strange, but I'll take a look at it when I get the chance and see if I can reproduce it.

**Comment by tgstyle** [comment]
I finally got to do some good troubleshooting with the player this seems to happen with the most. It turned out to be any weapon not just a Tinker's cross bow. So I ended up pulling mob rebirth from the server and it appears to have stopped the crashing. 

https://gist.github.com/tgstyle/626730749ce2751e5ce65680c40c1724

Could it be this line in the error report causing it?

"Entity's Vehicle: ~~ERROR~~ NullPointerException: null"

The biome the player is in is a BoP Bog.. It appears to happen most with mobs in water.
